src/lib/cli/format.ts

```diff
@@ -254,3 +254,38 @@ export const formatBytes = (
 export const formatMetricBytes = ( bytes: number, decimals = 2 ): string => {
 	return formatBytes( bytes, decimals, 1000 );
 };
+
+/*
+ * Get the duration between two dates
+ *
+ * @param {Date} from The start date
+ * @param {Date} to  The end date
+ * @returns {string} The duration between the two dates
+ */
+export function formatDuration( from: Date, to: Date ): string {
+	const millisecondsPerSecond = 1000;
+	const millisecondsPerMinute = 60 * millisecondsPerSecond;
+	const millisecondsPerHour = 60 * millisecondsPerMinute;
+	const millisecondsPerDay = 24 * millisecondsPerHour;
+
+	let duration = Math.abs( from.getTime() - to.getTime() );
+	if ( duration < 0 ) duration = 0;
```

Comment:
When `to` is much earlier than `from`, the `duration` returned is still positive. The problem with this is that if given a duration `from` and `to` returns `30 seconds` for `formatDuration`, after 35 seconds, it will return `5 seconds` left, despite us going 5 seconds over the expected duration.

To sum it up, we need to add a test case where `from` is greater than `to`. Is the value returned as expected?